library/src/main/java/com/alamkanak/weekview/DateTimeInterpreter.java

```diff
@@ -7,5 +7,5 @@
  */
 public interface DateTimeInterpreter {
     String interpretDate(Calendar date);
-    String interpretTime(int hour);
+    String interpretTime(int hour, int minutes);
```

Comment:
Maybe have a "subinterface"?

Because a breaking change for this small behaviour change is maybe not what we want.